observations/obs_converters/WOD/WOD.rst

```diff
@@ -0,0 +1,253 @@
+WOD Observations
+================
+
+Overview
+--------
+
+The World Ocean Database (WOD) is a collection of data from various sources,
+combined into a single format with uniform treatment. WOD is created by the 
+National Centers for Environmental Information (NCEI) of the National Oceanic
+and Atmospheric Administration (NOAA).
+
+An updated version of the dataset is released approximately every four years.
+It was first produced in 1994 and has been released in 1998, 2001, 2005, 2009,
+2013 and 2018.
+
+The `WOD website <https://www.ncei.noaa.gov/products/world-ocean-atlas>`__ has
+detailed information about the repository, observations, and datasets. The
+programs in this directory convert from the packed ASCII files found in the
+repository into DART observation sequence (obs_seq) file format.
+
+There are two sets of available files: the raw observations and the
+observations binned onto standard levels.
+
+.. note::
+
+   DAReS staff recommend using the datasets on standard levels for
+   assimilation. The raw data can be very dense in the vertical and are not
+   truly independent observations. The correlation between nearby observations
```

Comment:
Just want to make sure that everyone knows the assumption is that the observation *errors* need to be uncorrelated.